control-plane/agents/src/bin/core/controller/io_engine/mod.rs

```diff
@@ -156,6 +156,9 @@ pub(crate) trait ReplicaSnapshotApi {
         &self,
         request: &CreateReplicaSnapshot,
     ) -> Result<CreateReplicaSnapshotResp, SvcError>;
+
+    /// Remove a snapshot using the incoming id.
+    async fn delete_repl_snapshot(&self, request: &DeleteReplicaSnapshot) -> Result<(), SvcError>;
```

Comment:
Convention is `destroy`.
```suggestion
    async fn destroy_repl_snapshot(&self, request: &DestroyReplicaSnapshot) -> Result<(), SvcError>;
```

Reply:
Modified. The gRPC type name will remain DeleteReplicaSnapshotRequest though.

Reply:
Yeah, we need to rename the dataplane api.. for next time..